src/cpp/shared/resource/resourceManager.h

```diff
@@ -76,6 +76,8 @@ class resourceManager : baseManager
 
 extern resourceManager* gpResourceManager;
 
+extern void __fastcall WaitEndSample(struct SAMPLE2, int);
```

Comment:
Ok, so I remember that Sergei told me to put this specifically in resourceManager.h, but I know that this is a global variable and that I considered putting it in adv.h, before he told me to put it in resourceManager.h